collector/webui/webserver.go

```diff
@@ -53,9 +56,22 @@ func StartWebServer() {
 	app.RegisterView(tmpl)
 
 	go func() {
-		err := app.Listen(":8080")
+		err := app.Listen(":8081")
```

Comment:
It would be better to read from config file or command line rather than hard code.

The port should be user-defined to avoid port conflicts in user environments.